lib/console_manager.rb

```diff
@@ -79,12 +81,28 @@ def add_music
     on_spotify = gets.chomp.upcase
 
     # Action
-    @music_manager.add_music(publish_date, on_spotify == 'Y', archived == 'Y')
+    new_music = @music_manager.add_music(publish_date, on_spotify == 'Y', archived == 'Y')
+
+    add_genre(new_music)
 
     # Feedback
     puts "\nMusic Album has been registered successfully.\n\n"
   end
 
+  def add_genre(music)
+    # Add book label?
+    print 'Do you want to add a genre to this book? (Y/N)'
+    want_label = gets.chomp.upcase
+
+    return unless want_label == 'Y'
```

Comment:
I think you should rename **want_label** to **want_genre** to be consistent. Of course, this will not affect the logic.

Reply:
okays thanks